projects/jupyter-server-ydoc/jupyter_server_ydoc/handlers.py

```diff
@@ -105,6 +106,26 @@ async def prepare(self):
                         self._document_save_delay,
                     )
 
+                    def exception_logger(exception: Exception, log: Logger) -> bool:
+                        """A function that catches any exceptions raised in the websocket
+                        server and logs them.
+
+                        The protects the y-room's task group from cancelling
+                        anytime an exception is raised.
+                        """
+                        room_id = "unknown"
+                        if self.room.room_id:
```

Comment:
`self.room.room_id` can be `self._room_id`

Reply:
👍